--- ./Upload.php~	2005-05-27 13:02:42.000000000 +0300
+++ ./Upload.php	2005-05-27 13:02:52.000000000 +0300
@@ -560,7 +563,7 @@
         $this->HTTP_Upload_Error($lang);
         $ext = null;
 
-        if (empty($name) || $size == 0) {
+        if (empty($error) && (empty($name) || $size == 0)) {
             $error = 'NO_USER_FILE';
         } elseif ($tmp == 'none') {
             $error = 'TOO_LARGE';
