--- ./Upload.php~    2006-02-22 13:55:52.000000000 +0200
+++ ./Upload.php    2006-02-22 13:55:56.000000000 +0200
@@ -590,6 +590,13 @@
     var $_extensionsMode  = 'deny';
 
     /**
+     * @see HTTP_Upload_File::setValidExtensions()
+     * @var bool
+     * @access private
+     */
+     var $_extensionsCaseSensitive = true;
+
+    /**
      * Contains the desired chmod for uploaded files
      * @var int
      * @access private
@@ -934,12 +941,16 @@
      *                       1) 'deny'   Will deny only the supplied extensions
      *                       2) 'accept' Will accept only the supplied extensions
      *                                   as valid
+     * @param bool $case_sensitive Is extension check is case sensitive.
      * @access public
      */
-    function setValidExtensions($exts, $mode = 'deny')
+    function setValidExtensions($exts, $mode = 'deny', $case_sensitive = null)
     {
         $this->_extensionsCheck = $exts;
         $this->_extensionsMode  = $mode;
+        if (!is_null($case_sensitive)) {
+            $this->_extensionsCaseSensitive  = $case_sensitive;
+        }
     }
 
     /**
@@ -952,8 +963,12 @@
     {
         $exts = $this->_extensionsCheck;
         settype($exts, 'array');
+        $ext = $this->getProp('ext');
+        if (!$this->_extensionsCaseSensitive) {
+            $ext = strtolower($ext);
+        }
         if ($this->_extensionsMode == 'deny') {
-            if (in_array($this->getProp('ext'), $exts)) {
+            if (in_array($ext, $exts)) {
                 return false;
             }
         // mode == 'accept'
