--- ./Upload.php.org	2006-02-22 14:31:55.000000000 +0200
+++ ./Upload.php	2006-02-22 14:32:04.000000000 +0200
@@ -106,7 +106,8 @@
                 'nl'    => "Het bestand is te groot, de maximale grootte is: $maxsize bytes.",
                 'fr'    => "Le fichier est trop gros. La taille maximum autoris&eacute;e est: $maxsize bytes.",
                 'it'    => "Il file &eacute; troppo grande. Il massimo permesso &eacute: $maxsize bytes.",
-                'pt_BR' => "Arquivo muito grande. O tamanho m&aacute;ximo permitido &eacute; $maxsize bytes."
+                'pt_BR' => "Arquivo muito grande. O tamanho m&aacute;ximo permitido &eacute; $maxsize bytes.",
+                'ru'    => "Файл слишком велик. Максимальный размер: $maxsize байт.",
                 ),
             'MISSING_DIR' => array(
                 'es'    => 'Falta directorio destino.',
@@ -116,7 +117,8 @@
                 'nl'    => 'Geen bestemmings directory.',
                 'fr'    => 'Le r&eacute;pertoire de destination n\'est pas d&eacute;fini.',
                 'it'    => 'Manca la directory di destinazione.',
-                'pt_BR' => 'Aus&ecirc;ncia de diret&oacute;rio de destino.'
+                'pt_BR' => 'Aus&ecirc;ncia de diret&oacute;rio de destino.',
+                'ru'    => 'Не указана директория загрузки.',
                 ),
             'IS_NOT_DIR' => array(
                 'es'    => 'El directorio destino no existe o es un fichero regular.',
@@ -126,7 +128,8 @@
                 'nl'    => 'De doeldirectory bestaat niet, of is een gewoon bestand.',
                 'fr'    => 'Le r&eacute;pertoire de destination n\'existe pas ou il s\'agit d\'un fichier r&eacute;gulier.',
                 'it'    => 'La directory di destinazione non esiste o &eacute; un file.',
-                'pt_BR' => 'O diret&oacute;rio de destino n&atilde;o existe ou &eacute; um arquivo.'
+                'pt_BR' => 'O diret&oacute;rio de destino n&atilde;o existe ou &eacute; um arquivo.',
+                'ru'    => 'Директории загрузки не существует, или это обычный файл.',
                 ),
             'NO_WRITE_PERMS' => array(
                 'es'    => 'El directorio destino no tiene permisos de escritura.',
@@ -136,7 +139,8 @@
                 'nl'    => 'Geen toestemming om te schrijven in de doeldirectory.',
                 'fr'    => 'Le r&eacute;pertoire de destination n\'a pas les droits en &eacute;criture.',
                 'it'    => 'Non si hanno i permessi di scrittura sulla directory di destinazione.',
-                'pt_BR' => 'O diret&oacute;rio de destino n&atilde;o possui permiss&atilde;o para escrita.'
+                'pt_BR' => 'O diret&oacute;rio de destino n&atilde;o possui permiss&atilde;o para escrita.',
+                'ru'    => 'У вас нет прав писать в директорию загрузки.',
                 ),
             'NO_USER_FILE' => array(
                 'es'    => 'No se ha escogido fichero para el upload.',
@@ -146,7 +150,8 @@
                 'nl'    => 'Er is geen bestand opgegeven om te uploaden.',
                 'fr'    => 'Vous n\'avez pas s&eacute;lectionn&eacute; de fichier &agrave; envoyer.',
                 'it'    => 'Nessun file selezionato per l\'upload.',
-                'pt_BR' => 'Nenhum arquivo selecionado para upload.'
+                'pt_BR' => 'Nenhum arquivo selecionado para upload.',
+                'ru'    => 'Вы не выбрали файл загрузки.',
                 ),
             'BAD_FORM' => array(
                 'es'    => 'El formulario no contiene method="post" enctype="multipart/form-data" requerido.',
@@ -160,7 +165,8 @@
                            ' method="post" enctype="multipart/form-data".',
                 'it'    => 'Il modulo HTML non contiene gli attributi richiesti: "'.
                            ' method="post" enctype="multipart/form-data".',
-                'pt_BR' => 'O formul&aacute;rio HTML n&atilde;o possui o method="post" enctype="multipart/form-data" requerido.'
+                'pt_BR' => 'O formul&aacute;rio HTML n&atilde;o possui o method="post" enctype="multipart/form-data" requerido.',
+                'ru'    => 'HTML-форма не содержит необходимых переметров: method="post" enctype="multipart/form-data".',
                 ),
             'E_FAIL_COPY' => array(
                 'es'    => 'Fallo al copiar el fichero temporal.',
@@ -170,7 +176,8 @@
                 'nl'    => 'Het tijdelijke bestand kon niet gekopieerd worden.',
                 'fr'    => 'L\'enregistrement du fichier temporaire a &eacute;chou&eacute;.',
                 'it'    => 'Copia del file temporaneo fallita.',
-                'pt_BR' => 'Falha ao copiar o arquivo tempor&aacute;rio.'
+                'pt_BR' => 'Falha ao copiar o arquivo tempor&aacute;rio.',
+                'ru'    => 'Ошибка при копировании временного файла.',
                 ),
             'E_FAIL_MOVE' => array(
                 'es'    => 'No puedo mover el fichero.',
@@ -179,7 +186,8 @@
                 'de'    => 'Datei kann nicht verschoben werden.',
                 'nl'    => 'Het bestand kon niet verplaatst worden.',
                 'fr'    => 'Impossible de d&eacute;placer le fichier.',
-                'pt_BR' => 'N&atilde;o foi poss&iacute;vel mover o arquivo.'
+                'pt_BR' => 'N&atilde;o foi poss&iacute;vel mover o arquivo.',
+                'ru'    => 'Не удается переместить файл.',
                 ),
             'FILE_EXISTS' => array(
                 'es'    => 'El fichero destino ya existe.',
@@ -189,7 +197,8 @@
                 'nl'    => 'Het doelbestand bestaat al.',
                 'fr'    => 'Le fichier de destination existe d&eacute;j&agrave;.',
                 'it'    => 'File destinazione gi&agrave; esistente.',
-                'pt_BR' => 'O arquivo de destino j&aacute; existe.'
+                'pt_BR' => 'O arquivo de destino j&aacute; existe.',
+                'ru'    => 'Файл уже существует.',
                 ),
             'CANNOT_OVERWRITE' => array(
                 'es'    => 'El fichero destino ya existe y no se puede sobreescribir.',
@@ -199,7 +208,8 @@
                 'nl'    => 'Het doelbestand bestaat al, en kon niet worden overschreven.',
                 'fr'    => 'Le fichier de destination existe d&eacute;j&agrave; et ne peux pas &ecirc;tre remplac&eacute;.',
                 'it'    => 'File destinazione gi&agrave; esistente e non si pu&ograve; sovrascrivere.',
-                'pt_BR' => 'O arquivo de destino j&aacute; existe e n&atilde;o p&ocirc;de ser sobrescrito.'
+                'pt_BR' => 'O arquivo de destino j&aacute; existe e n&atilde;o p&ocirc;de ser sobrescrito.',
+                'ru'    => 'Файл уже сущесвтует и не может быть переписан.',
                 ),
             'NOT_ALLOWED_EXTENSION' => array(
                 'es'    => 'Extension de fichero no permitida.',
@@ -209,7 +219,8 @@
                 'nl'    => 'Niet toegestane bestands-extensie.',
                 'fr'    => 'Le fichier a une extension non autoris&eacute;e.',
                 'it'    => 'Estensione del File non permessa.',
-                'pt_BR' => 'Extens&atilde;o de arquivo n&atilde;o permitida.'
+                'pt_BR' => 'Extens&atilde;o de arquivo n&atilde;o permitida.',
+                'ru'    => 'Недопустимое расширение файла.',
                 ),
             'PARTIAL' => array(
                 'es'    => 'El fichero fue parcialmente subido',
@@ -217,7 +228,8 @@
                 'et'    => 'Faili &uuml;leslaadimine oli poolik.',
                 'de'    => 'Die Datei wurde unvollst&auml;ndig &uuml;bertragen.',
                 'nl'    => 'Het bestand is slechts gedeeltelijk geupload.',
-                'pt_BR' => 'O arquivo nгo foi enviado por completo.'
+                'pt_BR' => 'O arquivo nгo foi enviado por completo.',
+                'ru'    => 'Файл загружен лишь частично.',
                 ),
             'ERROR' => array(
                 'es'    => 'Error en subida:',
@@ -225,14 +237,16 @@
                 'et'    => 'Viga &uuml;leslaadimisel:',
                 'de'    => 'Fehler beim Upload:',
                 'nl'    => 'Upload fout:',
-                'pt_BR' => 'Erro de upload:'
+                'pt_BR' => 'Erro de upload:',
+                'ru'    => 'Ошибка загрузки:',
                 ),
             'DEV_NO_DEF_FILE' => array(
                 'es'    => 'No estб definido en el formulario este nombre de fichero como &lt;input type="file" name=?&gt;.',
                 'en'    => 'This filename is not defined in the form as &lt;input type="file" name=?&gt;.',
                 'et'    => 'Failinimi ei olnud vormistatud htmlis &lt;input type="file" name=?&gt;.',
                 'de'    => 'Dieser Dateiname ist im Formular nicht als &lt;input type="file" name=?&gt; definiert.',
-                'nl'    => 'Deze bestandsnaam is niett gedefineerd in het formulier als &lt;input type="file" name=?&gt;.'
+                'nl'    => 'Deze bestandsnaam is niett gedefineerd in het formulier als &lt;input type="file" name=?&gt;.',
+                'ru'    => 'Не указано имя файла в форме &lt;input type="file" name=?&gt;.',
                 )
         );
     }
