--- HTTP_Upload-1.0.0b1/usr/share/pear/HTTP/Upload.php~	2010-08-19 14:04:02.000000000 +0300
+++ HTTP_Upload-1.0.0b1/usr/share/pear/HTTP/Upload.php	2010-08-19 14:22:57.872355680 +0300
@@ -981,7 +981,7 @@
     {
         $this->_extensionsCheck = $exts;
         $this->_extensionsMode  = $mode;
-        if ($case_sensitive != null) {
+        if ($case_sensitive !== null) {
             $this->_extensionsCaseSensitive  = $case_sensitive;
         }
     }
